cfy_cluster_manager/main.py

```diff
@@ -598,17 +598,32 @@ def _validate_config_paths(existing_vms_list, using_three_nodes, errors_list):
                            'all instances or none of them.')
 
 
+def _validate_vms_not_deuplicated(existing_vms_dict, errors_list):
```

Comment:
typo: Should be duplicated, not deuplicated

Reply:
Right. Is there a tool to test for typos?